app/Brand.php

```diff
@@ -11,14 +11,15 @@ class Brand extends Model
 
     public $timestamps = false;
 
-    protected $fillable = ['name'];
+    protected $fillable = ['name', 'is_available'];
 
-    public static $columns = ['id', 'name'];
+    public static $columns = ['id', 'name', 'availability'];
 
     public static function form() : iterable
```

Comment:
what is this doing?

Reply:
rather than delete brands we could just render them active or inactive. It is boolean